themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fa.properties

```
@@ -0,0 +1,179 @@
+shareSuccess=منبع با موفقیت به اشتراک گذاشته شد.
+unShareSuccess=اشتراک‌گذاری منبع با موفقیت لغو شد.
+updateSuccess=منبع با موفقیت به روزرسانی شده.
+resourceAlreadyShared=منبع پیش از این با کاربر به اشتراک گذاشته شده بود.
+resourceNotShared=این منبع به اشتراک گذاشته نشده است.
+permissionRequests=درخواست‌های مجوز
+permissions=مجوزها
+selectPermissions=مجوز‌ها را انتخاب کنید
+unShareAllConfirm=آیا از حذف تمام اشتراک‌گذاری‌ها اطمینان دارید؟
+userNotFound=هیچ کاربری با نام یا ایمیل {0} یافت نشد
+
+# Linked accounts page
+linkedAccountsTitle=حساب‌های پیوندداده‌شده
+linkedAccountsIntroMessage=لاگین‌ها را از طریق حساب‌های شخص ثالث مدیریت کنید.
+linkedLoginProviders=ارائه‌دهندگان لاگین پیوند‌داده‌شده
+unlinkedLoginProviders=ارائه‌دهندگان لاگین بدون پیوند
+linkedEmpty=بدون ارائه‌دهنده پیوند‌داده‌شده
+unlinkedEmpty=بدون ارائه‌دهنده بدون پیوند
+socialLogin=لاگین شبکه‌‌‌‌های اجتماعی
+systemDefined=سیستم تعریف شد
+link=پیوند‌داده اکانت
+unLink=بدون پیوندسازی اکانت
+
+# Signing in page
+signingIn=در حال ورود
+signingInSubMessage=راه‌های ورود را تنظیم کنید.
+credentialCreatedAt=ایجادشده
+successRemovedMessage={0} حذف شد.
+stopUsingCred=متوقف‌کردن استفده از {0}.
+changePassword=تغییر رمز عبور
+removeCred=حذف {0}
+setUpNew=تنظیم {0}
+removeCredAriaLabel=حذف اعتبارنامه
+updateCredAriaLabel=به‌روزرسانی اعتبارنامه
+notSetUp={0} تنظیم نشده است.
+two-factor=اعتبارستجی دومرحله‌ای
+passwordless=بدون رمز عبور
+unknown=ناشناخته
+password-display-name=رمزعبور
+password-help-text=با واردکردن رمز عبور خود وارد شوید.
+password=رمز عبور من
+otp-display-name=اپلیکیشن اعتبارسنجی
+otp-help-text=یک کد از اپلیکیشن اعتبارسنجی وارد کنید.
+recovery-authn-code=کد‌های بازیابی اعتبارستجی من
+recovery-authn-codes-display-name=کدهای بازیابی اعبتارسنجی
+recovery-authn-codes-help-text=این کد‌ها میتوانند برای بازیابی مجدد دسترسی، در صورتی که سایر شیوه‌های اعتبار‌سنجی دو مرحله‌ای در دسترس نباشند، استفاده شوند.
+recovery-codes-number-used={0} از کدهای بازیابی استفاده شده
+recovery-codes-number-remaining={0} از کدهای بازیابی باقی مانده
+recovery-codes-generate-new-codes=برای دسترسی به حساب حود کدهای تازه اسجاد کنید
+webauthn-display-name=کلید امنیتی
+webauthn-help-text=از کلید امنیتی خود برای ورود استفاده کنید.
+webauthn-passwordless-display-name=کلید امنیتی
+webauthn-passwordless-help-text=ار کلید امنیتی خود برای ورود بدون رمز عبور استفاده کنید.
+basic-authentication=اعتبارسنجی پایه
+invalidRequestMessage=درخواست نامعتبر
+
+# Applications page
+applicationsPageTitle=اپلیکیشن‌ها
+applicationsPageSubTitle=مجوز‌های اپلیکیشن خود را مدیریت کنید
+internalApp=داخلی
+thirdPartyApp=شخص ثالث
+offlineAccess=دسترسی آفلاین
+inUse=در حال استفاده
+notInUse=آزاد
+applicationDetails=جزئیات اپلیکیشن
+client=مشتری
+description=توضیحات
+baseUrl=URL
+accessGrantedOn=دسترسی داده شد در
+removeButton=حذف دسترسی
+removeModalTitle=حذف دسترسی
+removeModalMessage=این عمل مجوز دسترسی اعطاشده فعلی برای {0} را حذف می‌کند. اگر می‌خواهید از این اپ دوباره استفاده کنید باید مجدد دسترسی را اعطا کنید.
```

```suggestion
removeModalMessage=با این کار مجوز دسترسی اعطا شده در حال حاضر برای {0} حذف خواهد شد. اگر می خواهید از این اپلیکیشن استفاده کنید، باید دوباره مجوز دسترسی بدهید.
```

Using the same translated message here as we had in this file: [js/apps/account-ui/public/locales/fa/translation.json](https://github.com/keycloak/keycloak/pull/19747/files#diff-572c71ad3aafccaceadd91c102a082fe63becb83163d878ee158dc11837c8b64)